README.md

```diff
@@ -73,52 +71,57 @@ You will need [Go](https://golang.org/) with `$GOPATH` defined and
 [yarn](https://yarnpkg.com/lang/en/docs/install/) installed.
 
 You need Protobuf 3. On Linux this might work:
-```
-> cd ~
-> wget https://github.com/google/protobuf/releases/download/v3.1.0/protoc-3.1.0-linux-x86_64.zip
-> unzip protoc-3.1.0-linux-x86_64.zip
-> export PATH=$HOME/bin:$PATH
+
+``` bash
+cd ~
+wget https://github.com/google/protobuf/releases/download/v3.1.0/protoc-3.1.0-linux-x86_64.zip
+unzip protoc-3.1.0-linux-x86_64.zip
+export PATH=$HOME/bin:$PATH
```

Comment:
Nice that people can copy/paste this script into their terminal directly! :tada: